---
output:
  bookdown::pdf_document2:
    latex_engine: xelatex
    toc: no
    number_sections: false
    keep_tex: true
    includes:
      in_header: "preamble.tex"
    citation_package: natbib
    extra_dependencies: ["float"]
bibliography: "references.bib"
biblio-style: "genetics"
title: A genetics-style document using RMarkdown
author:
header-includes:
- |
  ```{=latex}
  \usepackage{authblk}
  \author{Aaron P. Ragsdale \\ apragsdale@wisc.edu}
  \affil{Department of Integrative Biology, University of Wisconsin-Madison, USA}
  ```
- \usepackage{lineno}
- \linenumbers
date: \today
---

```{r setup, echo=FALSE}
```

\begin{abstract}

Most new mutations that affect fitness are deleterious and tend to be
eliminated from a population. The amount of time that a deleterious mutation
segregates depends on the strength of selection against genomes that carry it,
with very damaging mutations kept at low frequencies and purged relatively
rapidly. In the time between mutation and fixation or loss, selected variants,
both beneficial and damaging, can dramatically impact patterns of variation in
nearby linked regions. This distortion away from neutral expectations is now
empirically documented using sequencing data from an ever-growing set of study
systems, but questions remain about the primary mode of interactions between
multiple linked variants and their joint effects on genome-wide patterns of
diversity.

\end{abstract}

# Introduction

Most new mutations that affect fitness are deleterious and tend to be
eliminated from a population. The amount of time that a deleterious mutation
segregates depends on the strength of selection against genomes that carry it,
with very damaging mutations kept at low frequencies and purged relatively
rapidly. In the time between mutation and fixation or loss, selected variants,
both beneficial and damaging, can dramatically impact patterns of variation in
nearby linked regions. This distortion away from neutral expectations is now
empirically documented using sequencing data from an ever-growing set of study
systems, but questions remain about the primary mode of interactions between
multiple linked variants and their joint effects on genome-wide patterns of
diversity.

# Results

## Results subsection

Most new mutations that affect fitness are deleterious and tend to be
eliminated from a population. The amount of time that a deleterious mutation
segregates depends on the strength of selection against genomes that carry it,
with very damaging mutations kept at low frequencies and purged relatively
rapidly. In the time between mutation and fixation or loss, selected variants,
both beneficial and damaging, can dramatically impact patterns of variation in
nearby linked regions. This distortion away from neutral expectations is now
empirically documented using sequencing data from an ever-growing set of study
systems, but questions remain about the primary mode of interactions between
multiple linked variants and their joint effects on genome-wide patterns of
diversity.

Some results:

```{r fig1, echo=F, fig.cap="(ref:fig1cap)", fig.align="center", fig.pos="tbp!"}
knitr::include_graphics("figures/fig1.pdf")
```

(ref:fig1cap) \textbf{Figure 1 caption.} This is the caption for figure 1.

# Discussion

Previous theoretical and simulation studies have shown that interference and
interactions between selected mutations reduce the efficacy of selection at
linked loci, impacting substitution rates, the deleterious mutation load, and
dynamics of segregating mutations [@Hill1968-vu; @Birky1988-jm; @Barton1995-mj;
@McVean2000-ox].

# Methods

## Existing theory and numerical methods

Many well-known properties of two-locus dynamics and equilibrium LD come from
early work on the multi-locus diffusion approximation [@Kimura1955-qe;
@Hill1968-vu; @Ohta1969-ie; @Ohta1971-yd].

### Moment equation for $\Psi_n$

The system of linear ordinary differential equations for the entries of
$\Psi_n$ takes the form
\begin{equation}
(\#eq:system)
{\Psi}_n^{t+1}(i, j, k; t) = 
\mathcal{D}_{N(t)}\Psi_n^t
+ \mathcal{R}_{r}\Psi_{n+1}^t
+ \mathcal{U}_{u}\Psi_n^t
+ \mathcal{S}_{s_A, s_B, \ldots, h_A, h_B, \ldots}\Psi_{n+2}^t.
\end{equation}
Here, $\mathcal{D}_{N(t)}$ is a sparse linear operator accounting for drift
with population size $N(t)$, $\mathcal{R}$ accounts for recombination
with per-generation recombination probability $r$ between the two loci,
$\mathcal{U}$ accounts for mutation, either under an infinite sites or
biallelic reversible mutation model, and $\mathcal{S}$ accounts for
selection.

# References
